packages/e2e/src/factories.ts

```diff
@@ -34,6 +34,17 @@ import {
   Witnesser,
   util
 } from '@cardano-sdk/key-management';
+import {
```

Comment:
If the target of this PR is to have a server side blockfrost proxy, I would have expected no changes in the client... 🤔

Reply:
These are left-overs of the reverted old commit.
I removed client-side parts in 927c5c959ebebfb72a739564efa3603172425da0 .